lib/cw-core/src/tile.rs

```diff
@@ -276,7 +277,7 @@ impl Tile {
             return StatusCode::ErrorInternal;
         }
 
-        assert_eq!(
+        ma::assert_le!(
             frame_buffer.len(),
             255 * 5,
             "When using command_3_update_specific_led you cannot transfer more then 255 LED"
```

Comment:
This seems to be the bug, nice that you found it! However, you fix could be improved a little.

You do not need the `more_asserts` crate for this, you could rewrite the code to this, for the same functionality:

```rust
assert!(
    frame_buffer.len() <= (255 * 5),
    "When using command_3_update_specific_led you cannot transfer more then 255 LED"
)
```

Rust assert!() documentation if you are interested: https://doc.rust-lang.org/std/macro.assert.html